Meta/serenity.sh

```diff
@@ -171,7 +171,9 @@ pick_gcc() {
 
 cmd_with_target() {
     is_valid_target || ( >&2 echo "Unknown target: $TARGET"; usage )
-    pick_gcc
+    if [ "$TOOLCHAIN_TYPE" == "GNU" ]; then
+	pick_gcc
```

Comment:
Please indent using spaces

Reply:
Sorry, fixed indentation.

Reply:
This was only part of the issue. It now moves on to trying to build Lagom but like BertalanD said voidlinux's Clang is out of date so I can't do anything with Lagom.

Reply:
Since the issue is with my host system not having a suitable compiler for compiling Lagom I'm considering this part as resolved. This fix does as it is intended. The build moves on to trying to build Lagom.